### DISCUSSION TITLE
Looking for some help with getting orbital-sync up and running on Docker with Synology

### DISCUSSION THREAD

**Discussion by chrisray1997** [mattwebbio/orbital-sync#175] [Q&A]
I have one Pi-hole running on a bare metal Raspberry pi which I am using as my primary server and just installed a secondary docker image for Pi-hole to run as my secondary server. 

I installed the container, passed through the environment parameters for my primary and secondary servers however the container kept crashing and restarting. 

I am assuming I may have missed something on the initial setup so curious if anyone else was able to get orbital-sync up and running on a Synology using the Docker UI.

**Comment by mattwebbio** [comment]
Any chance you could share the log output? There could be a variety of reasons it's failing, so the logs are the best way for me to see what's happening. Setting the `VERBOSE` env var to `true` will make those logs more detailed, if you haven't done that already 🙂

**Reply by chrisray1997** [reply]
Well... I ripped out the container and started fresh and it seems to be working however it looks like it can't access the secondary Pi-hole running on the container.

This is the error I am seeing in the log.

login failed, reason: connect EHOSTUNREACH 192.168.4.57:80

[orbital-sync (1).csv](https://github.com/mattwebbio/orbital-sync/files/11277912/orbital-sync.1.csv)

**Reply by mattwebbio** [reply]
You may not be exposing port 80 on the secondary Pi-hole container, and if you were it likely wouldn't be on port 80 _externally_ because Synology probably uses this. I'd check your port mapping for the container first 🙂

**Reply by chrisray1997** [reply]
This issue seems to be related to the networking with my containers. The container for orbital-sync did not have access to the pi-hole container. 

After running the container on my PC, it works flawlessly. 

Thanks for the help!
